packages/docs/src/components/page-configs/ui-elements/va-date-picker/page-config.ts

```diff
@@ -1,7 +1,7 @@
-import { ApiDocsBlock } from '../../../../types/configTypes'
+import { ApiDocsBlock } from '@/types/configTypes'
 import VaDatePicker from 'vuestic-ui/src/components/va-date-picker/VaDatePicker.vue'
 import apiOptions from './api-options'
-import { DocsHelper } from '../../../../helpers/DocsHelper'
+import { DocsHelper } from '@/helpers/DocsHelper'
```

Comment:
We usually don't use a "@" alias and prefer relative path import. The "@" alias is only specific for webpack and it's usually breaking auto-imports.
```suggestion
import { DocsHelper } from '../../../../helpers/DocsHelper'
```